3rdParty/fuerte/src/types.cpp

```diff
@@ -23,6 +23,7 @@
 #include <fuerte/types.h>
 
 #include <algorithm>
+#include <stdexcept>
```

Comment:
This is required for GCC-10 compilation. It is used for `std::logic_error`. Which requires `stdexcept` by the standard.